### DISCUSSION TITLE
Support for storybook.js?

### DISCUSSION THREAD

**Discussion by parrotcar00** [varabyte/kobweb#622] [Q&A] [answered]
Does kobweb work with storybook (https://storybook.js.org/)?
or something which is Kotlin focused like: https://github.com/Kotlin/Storytale ?

**Comment by bitspittle** [comment, marked as answer]
Kobweb still early days so all focus is just getting the API surface complete and documented. It will be a long time I think before we have anything like UI editing tools.

As for Storytale, that looks like it's more important for cross platform development? Kobweb is really just focused on web development.

The main way to develop in Kobweb at the moment is to write your UIs in code, have a browser owned on the side, and keep a Kobweb server running. At that point, use the actual browser to preview your changes.
